scheduling/Shortest_Job_First Algorithm.py

```diff
@@ -0,0 +1,203 @@
+    complete = 0
+    increment_time = 0
+    minm = 999999999
+    short = 0
+    check = False
+
+    # Process until all processes gets 
+    # completed 
+    while (complete != no_of_processes):
+        for j in range(no_of_processes):
+            if(arrival_time[j]<=increment_time and(remaining_time[j]>0 and remaining_time[j]<minm)):
+                minm=remaining_time[j]
+              
+                short=j
+                check=True
+                
+        if (check == False): 
+            increment_time += 1
+            continue
+        remaining_time[short]-=1
+        
+        minm = remaining_time[short] 
+        if (minm == 0): 
+            minm = 999999999
+        
+        if (remaining_time[short] == 0): 
+
+ 
+            complete += 1
+            check = False
+
+            # Find finish time of current 
+            # process 
+            finish_time = increment_time + 1
+
+            # Calculate waiting time 
+            waiting_time[short] = (finish_time - arrival_time[short] -	
+                                burst_time[short]) 
+
+            if (waiting_time[short] < 0): 
+                waiting_time[short] = 0
+        
+        # Increment time 
+        increment_time += 1
+    return waiting_time
+def calculate_TurnAroundTime(burst_time, no_of_processes, waiting_time): 
+    """
+    This function calculates the Turn Around Times of each Processes
+        Return: list of Turn Around Time.
+    >>> calculate_TurnAroundTime([3,3,5,1], 4, [0,3,5,0])
+    [3, 6, 10, 1]
+    >>> calculate_TurnAroundTime([3,3], 2, [0,3])
+    [3, 6]
+    >>> calculate_TurnAroundTime([8,10,1], 3, [1,0,3])
+    [9, 10, 4]
+    """
+    turn_around_time=[0]*no_of_processes
+    for i in range(no_of_processes): 
+        turn_around_time[i] = burst_time[i] + waiting_time[i] 
+    return turn_around_time
+def calculate_average_times(waiting_time,turn_around_time, no_of_processes): 
+    """
+    This function calculates the average of the waiting & turnaround times
+        Prints: Average Waiting time & Average Turn Around Time
+    >>> calculate_average_times([0,3,5,0],[3,6,10,1],4)
+    Average waiting time = 2.00000 
+    Average turn around time =  5.0
+
+    >>> calculate_average_times([2,3],[3,6],2)
+    Average waiting time = 2.50000 
+    Average turn around time =  4.5
+
+    >>> calculate_average_times([10,4,3],[2,7,6],3)
+    Average waiting time = 5.66667 
+    Average turn around time =  5.0
+    """
+    total_waiting_time = 0
+    total_turn_around_time = 0
+    for i in range(no_of_processes): 
+
+        total_waiting_time = total_waiting_time + waiting_time[i] 
+        total_turn_around_time = total_turn_around_time + turn_around_time[i] 
+
+
+    print("\nAverage waiting time = %.5f "%(total_waiting_time /no_of_processes) ) 
+    print("Average turn around time = ", total_turn_around_time / no_of_processes) 
+    
+
+print("Enter How Many Process You want To Enter")
+no_of_processes=int(input())
```

Comment:
This line will fail our automated testing as discussed in CONTRIBUTING.md.

Reply:
Thanks a lot. I'm new to these flake Rules